docs/guides/flavors/ios.md

```diff
@@ -0,0 +1,190 @@
+---
+sidebar_position: 2
+title: iOS | Flavors
+sidebar_label: 🍎 iOS
+description: Push patches to multiple deployment tracks on iOS
+---
+
+# Flavors
+
+This guide will walk through how to setup an app in which there are 2 deployment tracks: `internal` and `stable`. It will cover how to validate a patch on the internal track and then promote the patch to the stable track on iOS.
+
+## Prerequisites
+
+This guide assumes the Shorebird CLI is installed on your machine and that you are logged into an account. Refer to the [getting started](/) instructions for more information.
+
+## Create a Project
+
+:::info
+In this guide, we'll go through the process of creating a new project from scratch. To apply these changes to an existing project, skip this step and read ahead.
+:::
+
+Create a new project using `flutter create multiple_deployments`.
+
+## Configure Flavors
+
+Next, edit the iOS project to support two flavors: `internal` and `stable` by following the instructions in the [Flutter documentation](https://docs.flutter.dev/deployment/flavors#creating-flavors-in-ios).
+
+## Initialize Shorebird
+
+Next, initialize shorebird in the current project via `shorebird init`.
+
+```
+shorebird init
+```
+
+The generated `shorebird.yaml` should look something like:
+
+```yaml
+# This file is used to configure the Shorebird updater used by your application.
+# Learn more at https://shorebird.dev
+# This file should be checked into version control.
```

Comment:
We should add a note in `shorebird init` output that these are not secrets.  People still get confused by that.

Reply:
https://github.com/shorebirdtech/shorebird/issues/969